src/deluge/storage/file_item.cpp

```diff
@@ -36,6 +38,16 @@ int32_t FileItem::setupWithInstrument(Instrument* newInstrument, bool hibernatin
 	isFolder = false;
 	instrumentAlreadyInSong = !hibernating;
 	displayName = filename.get();
+	String tempFilePath;
+	tempFilePath.set(newInstrument->dirPath.get());
+	tempFilePath.concatenate("/");
+	tempFilePath.concatenate(filename.get());
+	bool fileExists = storageManager.fileExists(tempFilePath.get(), &filePointer);
+	if (!fileExists) {
+		Debug::print("couldn't get filepath for file");
+		Debug::println(filename.get());
+		return false;
```

Comment:
Shouldn't this rather be:

```suggestion
		return ERROR_FILE_NOT_FOUND;
```